### ISSUE TITLE
[Mulgore] Problems with kodos' pathfinding, numbers and types.

### ISSUE THREAD

**Issue by Rorschach91** [azerothcore/azerothcore-wotlk#19652]
Bug Triager Notes: 

Original CC Report: https://github.com/chromiecraft/chromiecraft/issues/5484#issue-1680051674

I made a comparison between Chromiecraft Ptr and Cataclysm Classic Ptr.

In Chromiecraft:

1. Many Kodo's Packs are not following a path. They stay restricted in an area.
2. Some Kodo's Packs have not currect npc types (or they have too many npcs).
3. I haven't be able to find a particular Kodo's Packs that exist in CC, but not in Cata ptr.

To make a comparison between the packs, I leave some videos that I recorded on both servers.

Pack 1: [Cata](https://youtu.be/RKENdHm2Du4)/  [CC](https://youtu.be/2rlhOcogArs)
Pack 2: [Cata](https://youtu.be/XRZ2HvXfwDs)  /  [CC](https://youtu.be/4s6uaLaKa6c)
Pack 3: [Cata](https://youtu.be/ysNH2ep0Sus)  /  [CC](https://youtu.be/VKJc_fqckKw)
Pack 4 (unable to find on Cata): [CC](https://youtu.be/KS5qwD_Rg4M)

Edit: I checked in Wow Classic Era (Vanilla) and seems there are some differences:

1. Kodo Bulls can be found walking alone on the map.
2. Kodo's Packs can be composed by Matriarchs and Calfs.
3. Kodo Bull may not be present in the packs.

![World Of Warcraft - Classic Era Screenshot 2024 08 17 - 15 01 37 43](https://github.com/user-attachments/assets/a74afce3-9470-4992-8cb4-0592cbd5d3f8)
![World Of Warcraft - Classic Era Screenshot 2024 08 17 - 14 59 02 76](https://github.com/user-attachments/assets/9e345157-cfda-4083-a8c8-00a2270ea493)
![World Of Warcraft - Classic Era Screenshot 2024 08 17 - 15 00 39 52](https://github.com/user-attachments/assets/ed24524d-c216-4c60-bf01-12c741190b27)
![World Of Warcraft - Classic Era Screenshot 2024 08 17 - 15 01 06 39](https://github.com/user-attachments/assets/8e0f335e-c7dd-4db9-b8d3-80aff424bb67)

My Final Consideration: That Kodos have problems on CC is a fact, but I don't know how the fixers should move.
Unfortunately, with the changeover to Cataclysm Classic, we no longer have a reliable source. 

### What client do you play on?

enUS

### Faction

Horde

### Content Phase:

Generic

### Current Behaviour

Kodo bulls have no paths - they are not followed by the other smaller kodos

### Expected Blizzlike Behaviour

Kodo bulls walks around Mulgore with 2-3 other Kodo Matriarchs

### Source

https://wowpedia.fandom.com/wiki/Kodo_Bull

### Steps to reproduce the problem

Teleport to MUlgore and find Kodo Bulls
Observe behaviour

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/ce6ca597b54e64f328cacddae5f003cdc41a343d

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Grimdhex** [comment]
Just a note, it will need to get from classic era, and not cata since the revamp of kalimdor and eastern kingdom in extension.

**Comment by Rorschach91** [comment]
I don't think that mulgore has changed soo much during wotlk/cata change. Anyway I can try to check on vanilla server, but I've the feeling that It's the same.
